headerCheck/headerCheckFactory.go

```diff
@@ -42,6 +42,8 @@ func NewHeaderCheckHandler(proxy Proxy) (HeaderVerifier, error) {
 		return nil, err
 	}
 
+	multiSignerContainer := disabled.NewMultiSignerContainer()
```

Comment:
here we need a real multi signer container which includes cryptoComp.MultiSig

Reply:
changed :+1: